tcms/core/tests/test_middleware.py

```diff
@@ -12,11 +14,33 @@
 class TestCheckUnappliedMigrationsMiddleware(test.TransactionTestCase):
     def test_unapplied_migrations(self):
         call_command("migrate", "bugs", "zero", verbosity=2, interactive=False)
-        unapplied_migration_message = (
+        unapplied_migration_message = _(
             "unapplied migration(s). See "
             '<a href="https://kiwitcms.readthedocs.io/en/latest/'
             "installing_docker.html#initial-configuration-of-running-"
             'container">documentation</a>'
         )
         response = self.client.get("/", follow=True)
         self.assertContains(response, unapplied_migration_message)
+
+
+class TestCheckSettingsMiddleware(test.TestCase):
+    @classmethod
+    def setUpClass(cls):
+        super().setUpClass()
+        cls.base_url_error_message = _(
+            "Base URL is not configured! See "
+            '<a href="https://kiwitcms.readthedocs.io/en/latest/admin.html'
+            '#configure-kiwi-s-base-url">documentation</a> and <a href="'
+            '/admin/sites/site/1/change/">change it</a>!'
```

Comment:
String format should be exactly the same as in middleware.py in order for the translation to work.